docs/conf.py

```diff
@@ -15,155 +15,156 @@
 extensions = [
-    'sphinx.ext.imgmath',
-    'sphinx.ext.todo',
-    'sphinx.ext.intersphinx',
-    'sphinx.ext.autodoc',
-    'sphinx_issues',  # if this is missing, pip install sphinx-issues
-    'sphinx_tabs.tabs',  # if this is missing, pip install sphinx-tabs
-    ]
-
-try: import rst2pdf
-except ImportError: pass
-else: extensions.append('rst2pdf.pdfbuilder')
+    "sphinx.ext.imgmath",
+    "sphinx.ext.todo",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.autodoc",
+    "sphinx_issues",  # if this is missing, pip install sphinx-issues
+    "sphinx_tabs.tabs",  # if this is missing, pip install sphinx-tabs
+    "sphinx_rtd_theme",
+]
+
+# try: import rst2pdf
+# except ImportError: pass
+# else: extensions.append('rst2pdf.pdfbuilder')
 
 # Add any paths that contain templates here, relative to this directory.
-templates_path = ['_templates']
+# templates_path = ['_templates']
 
 # The suffix of source filenames.
-source_suffix = '.rst'
+source_suffix = ".rst"
 
 # The encoding of source files.
-#source_encoding = 'utf-8-sig'
+# source_encoding = 'utf-8-sig'
 
 # The master toctree document.
-master_doc = 'index'
+master_doc = "index"
 
 # General information about the project.
-project = 'Cython'
-authors = 'Stefan Behnel, Robert Bradshaw, Dag Sverre Seljebotn, Greg Ewing, William Stein, Gabriel Gellner, et al.'
-copyright = '%s, %s' % (YEAR, authors)
+project = "Cython"
+authors = "Stefan Behnel, Robert Bradshaw, Dag Sverre Seljebotn, Greg Ewing, William Stein, Gabriel Gellner, et al."
+copyright = f"{YEAR, authors}"
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The full version, including alpha/beta/rc tags.
-release = '0.15'
-try:
-    _match_version = re.compile(r'^\s*_*version\s*_*\s*=\s*["\']([^"\']+)["\'].*').match
-    with open(os.path.join(os.path.dirname(__file__), '..', 'Cython', 'Shadow.py')) as _f:
-        for line in itertools.islice(_f, 5):  # assume version comes early enough
-            _m = _match_version(line)
-            if _m:
-                release = _m.group(1)
-                break
-        else:
-            print("FAILED TO PARSE PROJECT VERSION !")
-except:
-    pass
-# The short X.Y version.
-version = re.sub('^([0-9]+[.][0-9]+).*', '\g<1>', release)
+# release = '0.15'
+# try:
+#     _match_version = re.compile(r'^\s*_*version\s*_*\s*=\s*["\']([^"\']+)["\'].*').match
+#     with open(os.path.join(os.path.dirname(__file__), '..', 'Cython', 'Shadow.py')) as _f:
+#         for line in itertools.islice(_f, 5):  # assume version comes early enough
+#             _m = _match_version(line)
+#             if _m:
+#                 release = _m.group(1)
+#                 break
+#         else:
+#             print("FAILED TO PARSE PROJECT VERSION !")
+# except:
+#     pass
+# # The short X.Y version.
+# version = re.sub('^([0-9]+[.][0-9]+).*', '\g<1>', release)
```

Comment:
Why did you remove this part?

Reply:
This is my bad, when build locally I got errors so try to comment block that I don't understand